prio-graph-scheduler/src/read_write_account_set.rs

```diff
@@ -0,0 +1,287 @@
+use {
+  ahash::AHashSet,
+  solana_sdk::{message::SanitizedMessage, pubkey::Pubkey},
+};
+
+/// Wrapper struct to accumulate locks for a batch of transactions.
+#[derive(Debug, Default)]
```

Comment:
have no idea about where to public this type. Since the usage on scheduler is only [here](https://github.com/anza-xyz/agave/pull/3167/files#diff-adc96c136d703b8bf07267822eaa7a1accb8ae59ed033d918f5e9676fc0a4ab3R96), i just copy one now.